### DISCUSSION TITLE
Non-compliant input components

### DISCUSSION THREAD

**Discussion by jlee-enquizit** [trussworks/react-uswds#2264] [Q&A]
There seem to be inconsistencies with input error-handling between trussworks components USWDS standards.
I've opened a bug here: https://github.com/trussworks/react-uswds/issues/2263
Is this something that our team should tackle or should we wait for an update to bring react-uswds inline with USWDS 3.1?

Thanks!
JL

**Comment by brandonlenz** [comment]
Hello @jlee-enquizit the ability to add a validationStatus prop for DatePicker (as well as Dropdown/Select) will be included in the 5.0.0 release!
